multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -283,14 +283,16 @@ def create_transaction_for_setting_special_role_on_fungible_token(
         user: IAddress,
         token_identifier: str,
         add_role_local_mint: bool,
-        add_role_local_burn: bool
+        add_role_local_burn: bool,
+        add_role_esdt_transfer_role: bool
```

Comment:
Breaking change, but we can live with it (there will be a minor version anyway).